SingleSampleQc.wdl

```diff
@@ -100,20 +102,37 @@ workflow WholeGenomeSingleSampleQc {
       metrics_filename = base_name + ".quality_yield_metrics",
       preemptible_tries = preemptible_tries
   }
-
-  # QC the sample raw WGS metrics (common thresholds)
-  call QC.CollectRawWgsMetrics as CollectRawWgsMetrics {
-    input:
-      input_bam = BuildBamIndex.bam,
-      input_bam_index = BuildBamIndex.bam_index,
-      metrics_filename = base_name + ".raw_wgs_metrics",
-      ref_fasta = ref_fasta,
-      ref_fasta_index = ref_fasta_index,
-      wgs_coverage_interval_list = wgs_coverage_interval_list,
-      read_length = read_length,
-      preemptible_tries = preemptible_tries
+  
+  if (is_wgs) {
+    # QC the sample raw WGS metrics since this IS WGS data
+    call QC.CollectRawWgsMetrics as CollectRawWgsMetrics {
+      input:
+        input_bam = BuildBamIndex.bam,
+        input_bam_index = BuildBamIndex.bam_index,
+        metrics_filename = base_name + ".raw_wgs_metrics",
+        ref_fasta = ref_fasta,
+        ref_fasta_index = ref_fasta_index,
+        wgs_coverage_interval_list = coverage_interval_list,
+        read_length = read_length,
+        preemptible_tries = preemptible_tries
+    }
   }
-
+  
+  if (!is_wgs) {
+    # QC the sample WES metrics since this IS NOT WGS
+    call QC.CollectHsMetrics as CollectHsMetrics {
+      input:
+        input_bam = BuildBamIndex.bam,
+        input_bam_index = BuildBamIndex.bam_index,
+        ref_fasta = ref_fasta,
+        ref_fasta_index = ref_fasta_index,
+        metrics_filename = base_name + ".hs_metrics",
+        target_interval_list = coverage_interval_list,
+        bait_interval_list = coverage_interval_list,
```

Comment:
Is it okay using the same file for both of these?

Reply:
in testing this worked.

Reply:
Makes sense that it would "work" in that it runs.  Mostly thinking it might make some of the results less meaningful.